demo/api/schema.gql

```diff
@@ -105,6 +92,21 @@ enum LicenseType {
   RIGHTS_MANAGED
 }
 
+type CurrentUserPermission {
+  permission: String!
+  contentScopes: [JSONObject!]
+}
+
+type CurrentUser {
+  id: String!
+  name: String!
+  email: String!
+  language: String!
+  allowedContentScopes: [JSONObject!]!
+  contentScopes: [JSONObject!]
```

Comment:
Why do we need both here? Can't allowedContentScopes be `null` as well?

Reply:
So `contentScopes` are the content scopes a user has (i.e. owns), and `allowedContentScopes` are `contentScopes`+ some scopes which a user has been given access to using user permissions, is this correct?

Reply:
Thanks for the explanation! Could we add this either as comment in code, or as description to the GraphQL fields? I believe we'll otherwise discuss this again and again...